agent/gpu/nvidia_gpu_manager_unix.go

```diff
@@ -44,7 +44,7 @@ type NvidiaGPUManager struct {
 	DriverVersion string                `json:"DriverVersion"`
 	GPUIDs        []string              `json:"GPUIDs"`
 	GPUDevices    []*ecs.PlatformDevice `json:"-"`
-	lock          sync.RWMutex          `json:"-"`
```

Comment:
`agent/gpu/nvidia_gpu_manager_unix.go:47: struct field lock has json tag but is not exported`